app/scripts/move-popup/dimMoveItemProperties.directive.js

```diff
@@ -186,5 +154,69 @@
         });
       }
     }
+
+    /*
+     * Determins if the item has damange and it's non kinetic.
+     *
+     * @param {Object} item - A item from the destiny server
```

Comment:
This isn't a terribly helpful description (nor is it correct - this is a DIM item that has been processed and built by us, not the raw info from Bungie)